### PR TITLE
#8 connect firebase to the project

### LINKED ISSUE #8: Connect firebase to the project
- [x] Connect to firebase
- [x] Add realtime database

### PR DESCRIPTION
**Pull request by kavusiks**
Pull request for: https://github.com/WilliamHLe/star-battle-ntnu/projects/2#card-79017085

### CODE REVIEW

File: android/src/group16/project/game/AndroidFirebaseConnection.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    override fun setValueInDb(target: String, value: String) {
+        //TODO("Not yet implemented")
+        var myRef=database.getReference(target)
+        if(myRef != null) {
+            myRef.setValue(value)
+        } else {
+            println("Databaseref was not found.")
+        }
+    }
```
Comment (WilliamHLe) [inline android/src/group16/project/game/AndroidFirebaseConnection.kt:20]: What is this suppose to do?

Comment (WilliamHLe) [inline android/src/group16/project/game/AndroidFirebaseConnection.kt:20]: What is this suppose to do?

Comment (WilliamHLe) [inline android/src/group16/project/game/AndroidFirebaseConnection.kt:20]: What is this suppose to do?

**Review by WilliamHLe — APPROVED** [review]
Looks good to me, just comment the code more often!

File: android/src/group16/project/game/AndroidFirebaseConnection.kt (hunk 1 above)
Comment (kavusiks) [reply android/src/group16/project/game/AndroidFirebaseConnection.kt:20]: to add values to our realtime db by given reference

**Pull request merged**